src/SupervisionSuccess/components/Chart/Chart.js

```diff
@@ -141,8 +141,10 @@ const Chart = ({ isError, data, startYear, isNotAvailable2020 }) => {
               padding: 20,
               fontSize: isMobile ? 12 : 16,
               precision: 0,
-              suggestedMax: min === max ? max + Y_AXIS_OFFSET : max + (max - min) * VERTICAL_OFFSET,
-              suggestedMin: min === max ? max - Y_AXIS_OFFSET : min - (max - min) * VERTICAL_OFFSET,
+              suggestedMax:
+                maxY > minY ? maxY + (maxY - minY) * VERTICAL_OFFSET : minY + Y_AXIS_OFFSET,
```

Comment:
I don't love this approach for two reasons:
1. It kind of elides the interesting question of _why min would ever be higher than max_? It's fairly surprising to the reader, and the fact that we have a unit test that shows such a result means that future readers will think that we thought it was totally reasonable and be confused. Those two numbers ultimately get calculated in `produceProjections` where we pass in +50 to one and -50 to the other, holding everything else constant, so I find it unexpected and would prefer that we either figure out why it can happen and document that in a comment somewhere nearby, if possible.
2. It tries to solve for that problem with an arbitrary padding for which there's no clear reason to expect that min would always be within that threshold from max when it is higher. Why 1000 and not 100 or 10,000 or 100,000?

With that in mind, I wonder if (besides trying to figure out why this can happen), the right solution would be down in  `produceProjections` to calculate the max and min sums are you currently do, but then set `max` to the max of those two sums and `min` to the min of those two sums? That way, we're saying, "Regardless of what comes out of these projections, we're calling max the higher of the two values and min the lower," and then by the time we are handling max and min up here, we can assume that they're properly ordered.

Reply:
Good suggestion. I resolved this with your approach, will try to figure out later why this strange behavior could happen in tests.